tensorflow/compiler/tf2tensorrt/convert/ops/einsum.cc

```diff
@@ -729,6 +732,192 @@ class ConvertEinsum : public OpConverterBase<ConvertEinsum> {
   std::unique_ptr<EinsumDescriptor> descriptor_a{nullptr};
   std::unique_ptr<EinsumDescriptor> descriptor_b{nullptr};
 };
+#else
+
+// Helper class to reindex equations to contain only lowercase characters.
```

Comment:
Would you please document these:
   Assumption on the input string, ReIndexer(eq), if there are such assumption.
   How do we reindex equations to contain only lowercase chars.